packages/network/nss-mdns/package.mk

```diff
@@ -29,9 +29,8 @@ PKG_SHORTDESC="nss-mdns is a plugin for nss to allow name resolution via Multica
 PKG_LONGDESC="nss-mdns is a plugin for the GNU Name Service Switch (NSS) functionality of the GNU C Library (glibc) providing host name resolution via Multicast DNS"
 PKG_TOOLCHAIN="autotools"
 
-makeinstall_target() {
-  mkdir -p $SYSROOT_PREFIX/usr/lib
-    cp -P $PKG_BUILD/.$TARGET_NAME/src/.libs/libnss_mdns_minimal.so.2 $SYSROOT_PREFIX/usr/lib
```

Comment:
> Avahi isn't loading libnss_mdns_minimal.so.

The bug in the old code is copying the lib to $SYSROOT_PREFIX only omitting $INSTALL.

But I agree with you the few kB don't harm.

Reply:
Yes absolutely! However even if the minimal library had been installed avahi wouldn't have loaded it.